public/pages/Detectors/containers/Detector/DetectorDetails.tsx

```diff
@@ -338,11 +339,9 @@ export class DetectorDetails extends React.Component<DetectorDetailsProps, Detec
                 </EuiTitle>
               </EuiFlexItem>
               <EuiFlexItem grow={false}>
-                <EuiText>
-                  <span style={{ color: `${detector.enabled ? 'green' : 'red'}` }}>
-                    {detector.enabled ? 'Active' : 'Inactive'}
-                  </span>
-                </EuiText>
+                <EuiHealth color={detector.enabled ? 'subdued' : 'primary'}>
```

Comment:
Shouldn't it use primary when the detector is enabled?

Reply:
In addition to Amardeep's question, Ming mentioned we should use `success` for the `active` color 
https://github.com/opensearch-project/security-analytics-dashboards-plugin/issues/143#issuecomment-1320702495